src/bookshelf/src/bksw_wm_sql_object.erl

```diff
@@ -0,0 +1,315 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
```

Comment:
These are the items left:

```
src/bksw_wm_sql_object.erl
213:                          %% TODO CHECK hash checksums/length before exiting successfully
288:                %% TODO: This can fail; we need some method for managing failure here.
323:    %% TODO: This can fail; we need some method for managing failure here.
```